packages/nimble-components/src/select/tests/select.spec.ts

```diff
@@ -281,16 +281,29 @@ describe('Select', () => {
         await disconnect();
     });
 
-    it('changing textContent of selected option updates display of select', async () => {
+    it('option added directly to DOM synchronously registers with Select', async () => {
```

Comment:
Was it intentional to remove the previous test `changing textContent of selected option updates display of select`?

Reply:
Don't think so, betting it was a merge issue, I re-added the test and tweaked the synchronous issue comments below